OWNERS_ALIASES

```diff
@@ -12,7 +12,7 @@ aliases:
 
   # https://github.com/kubernetes/sig-release/blob/master/release-managers.md#build-admins
   build-admins:
-    - amwat
+  # TODO: add juanfescobar after https://github.com/kubernetes/org/issues/3193
```

Comment:
is this the correct indentation level?

Reply:
AFAIK comments do not have meaningful indentation anyhow

Reply:
right, this is not yaml :)

Reply:
er it is yaml? the parser in prow is, but comment indent isn't meaningful. you can indent comment lines however you wish.